File: WBDrata/WEB-INF/classes/npackage/getdatamember.java
package npackage;

import java.io.*;
import javax.servlet.*;
import javax.servlet.http.*;
/**
 * Servlet implementation class LoginServlet
 */
public class getdatamember extends HttpServlet {


@Override
public void doGet(HttpServletRequest request,
				  HttpServletResponse response)
	throws IOException, ServletException
{
	doPost(request, response);
}
	   
@Override
public void doPost(HttpServletRequest request, HttpServletResponse response) 
	   throws ServletException, java.io.IOException {

try
{	    

	UserBean user = new UserBean();
	 
	user.setType("getdatamember");
	user.setQuery("SELECT * FROM  user WHERE username='"
                        + request.getSession().getAttribute("username")
                        + "'");
						
	
   user = UserDAO.login(user);
	 	
		boolean coba = user.isValid();
		
    if (user.isValid())
    {    
		HttpSession session = request.getSession(true);	    
        session.setAttribute("username",user.getUsername()); 
		session.setAttribute("password",user.getPassword()); 
		session.setAttribute("cardnumber",user.getNoCredit());
        session.setAttribute("email",user.getEmail()); 
		session.setAttribute("namalengkap",user.getNamaLengkap()); 
		session.setAttribute("nohp",user.getNoHP());
        session.setAttribute("provinsi",user.getProvinsi()); 
		session.setAttribute("kotakabupaten",user.getKotaKabupaten()); 
		session.setAttribute("alamat",user.getAlamat());	
		session.setAttribute("kodepos",user.getKodePos());	
		
		
		user.setType("getjumlahtransaksi");
		user.setQuery("SELECT count(*) AS COUNT FROM  transactionlog WHERE Username='"
                        + request.getSession().getAttribute("username")
                        + "'");
		user = UserDAO.login(user);

		session.setAttribute("jumlahtransaksi",user.getJumlahTransaksi());	
		
		response.sendRedirect("edit-profile.jsp");
    }
	       
    else 
       response.sendRedirect("index.jsp");


} 
		
		
catch (Throwable theException) 	    
{
     System.out.println(theException); 
}
       }
	   
	   
	   
	   
	   
	}

File: BOCI6/getdatamember.php
<?php  

session_start();

require_once("databaseconnect.php");   //memanggil file databaseconnect.php  

connect_db();       // memanggil fungsi connect_db yang adadi  file databaseconnect.php  

$username = $_COOKIE['username'];
$password = $_COOKIE['password'];

$data = mysql_query("SELECT * FROM user where username='$username'") 
 or die(mysql_error()); 
 $num=mysql_numrows($data);
$data3 = mysql_query("select count(*) from transactionlog where username='$username';");
$data2=mysql_fetch_array($data3);
$i=0;
while ($i < $num) {

$email=mysql_result($data,$i,"email");
$namalengkap=mysql_result($data,$i,"namalengkap");
$nohp=mysql_result($data,$i,"nohp");
$provinsi=mysql_result($data,$i,"provinsi");
$kotakabupaten=mysql_result($data,$i,"kotakabupaten");
$alamat=mysql_result($data,$i,"alamat");
$kodepos=mysql_result($data,$i,"kodepos");
$cardnumber=mysql_result($data,$i,"nocredit");
$i++;
}
$_SESSION['username']=$username;
$_SESSION['password']=$password;
$_SESSION['cardnumber']=$cardnumber;
$_SESSION['email']=$email;
$_SESSION['namalengkap']=$namalengkap;
$_SESSION['nohp']=$nohp;
$_SESSION['provinsi']=$provinsi;
$_SESSION['kotakabupaten']=$kotakabupaten;
$_SESSION['alamat']=$alamat;
$_SESSION['kodepos']=$kodepos;
$_SESSION['jumlahtransaksi']=$data2[0];

header('location: edit-profile.php');  

 ?> 